test/Microsoft.Azure.WebJobs.Host.UnitTests/Executors/DefaultStorageAccountProviderTests.cs

```diff
@@ -265,7 +265,7 @@ public void StorageAccountOverrides_MultipleLevels()
             DefaultStorageAccountProvider provider = CreateProductUnderTest(services, connectionStringProvider, parser, validator);
             var exception = await Assert.ThrowsAsync<InvalidOperationException>(() => provider.GetDashboardAccountAsync(CancellationToken.None));
 
-            Assert.Equal("Storage account 'name' is of unsupported type 'Premium'. Supported types are 'GeneralPurpose'", exception.Message);
+            Assert.Equal("Storage account 'name' is of unsupported type 'Premium'. Supported types are 'General Purpose'", exception.Message);
```

Comment:
nit: Can you have one test that uses multiple supported types? The message looks good, it'd just be nice to have a validation in there.